hack/generate-manifest.sh

```diff
@@ -144,6 +149,12 @@ if [[ $ENCAP_MODE != "" ]]; then
     sed -i.bak -E "s/^[[:space:]]*#[[:space:]]*trafficEncapMode[[:space:]]*:[[:space:]]*[a-z]+[[:space:]]*$/trafficEncapMode: $ENCAP_MODE/" antrea-agent.conf
 fi
 
+if [[ $CLOUD != "" ]]; then
+    if [[ $CLOUD == "GKE" ]]; then
+        sed -i.bak -E "s/^[[:space:]]*#[[:space:]]*trafficEncapMode[[:space:]]*:[[:space:]]*[a-z]+[[:space:]]*$/trafficEncapMode: noEncap/" antrea-agent.conf
```

Comment:
Not sure if we should reuse the `ENCAP_MODE` option instead as it can be provided at the same time as `CLOUD` so they might conflict, then the command above could be:
`$(CURDIR)/hack/generate-manifest.sh --mode dev --cloud GKE --encap-mode noEncap > build/yamls/antrea-gke.yml`

Reply:
i think the above suggestion makes it more cleaner

Reply:
Done